### PR TITLE
chore(demo): improve documentation for standalone components

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [x] Documentation content changes

### CODE REVIEW

File: projects/demo/src/modules/app/home/examples/main-standalone-optional.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
-import {TuiAlertModule, TuiDialogModule, TuiRootModule} from '@taiga-ui/core';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+import {TuiAlertModule, TuiDialogModule, TuiRootModule, TuiPushModule} from '@taiga-ui/core';
 
 bootstrapApplication(AppComponent, {
-  providers: [importProvidersFrom(TuiRootModule, TuiDialogModule, TuiAlertModule), provideAnimations()],
+  providers: [
+    importProvidersFrom(
+      BrowserAnimationsModule,
+      EventPluginsModule,
```
Comment (waterplea) [inline projects/demo/src/modules/app/home/examples/main-standalone-optional.md:9]: You do not need it, it's under the hood of root.

**Review by waterplea — COMMENTED** [review]
I'm not sure about it, isn't `provideAnimations` a preferred way to use animations now?
https://angular.io/guide/animations#step-1-enabling-the-animations-module

File: projects/demo/src/modules/app/home/examples/main-standalone-optional.md (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/app/home/examples/main-standalone-optional.md]: fixed

**Comment by splincode** [comment]
@waterplea fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**